app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def index
+    @users = User.order(id: 'DESC').page(params[:page]).per(10)
```

Comment:
```suggestion
    @users = User.order(id: :desc).page(params[:page]).per(10)
```

って書くことが多いかも？

https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-order

Reply:
確認しました。
小文字で書くのが一般的なんですね🙇‍♂️
https://api.rubyonrails.org/v7.0/classes/ActiveRecord/QueryMethods.html#method-i-order